### ISSUE TITLE
Tasks created in the build.gradle file should have the ability to call other tasks, rather than being executed only once depending on dependencies

### ISSUE THREAD

**Issue by notcaremath** [gradle/gradle#27377]
##### What if I want to add an auto-compile task to a gradle task to detect if the dependent file has changed to rebuild it, but I can't call the build task.
This is pseudocode
```
task("autoBuild") {
    doLast {
        println("autoBuild")
        var str = ""
        while (true) {
            val md5AsBase64 = Md5Utils.md5AsBase64(file("xxx.jar"))
            if (!StringUtils.equals(str, md5AsBase64)) {
                tasks.build.get().execute()
                str = md5AsBase64
            }
            Thread.sleep(5000)
        }
    }
}
```

### Current Behavior (optional)

Tasks can only be executed through dependencies

### Context

gradle 8.4

**Comment by ov7a** [comment]
Sorry that you're having trouble with Gradle!

Your issue appears to be a question about how to use Gradle. Your question might be answered in Gradle documentation or on our community Slack or forums. Find links to these resources on [help.gradle.org](https://help.gradle.org/).

If you think you have come across a Gradle bug, please file a new issue describing the problem and how to reproduce it.  

If you think Gradle is missing a feature, please file a new issue describing your use case for the feature.

***
This is already supported.

Please look at our documentation about [continuous build](https://docs.gradle.org/current/userguide/command_line_interface.html#sec:continuous_build) and [incremental build](https://docs.gradle.org/current/userguide/incremental_build.html#incremental_build).
